src/pretix/control/forms/item.py

```diff
@@ -272,6 +279,126 @@ class Meta:
         ]
 
 
+class QuestionFilterForm(forms.Form):
```

Comment:
I would name it `QuestionAnswerFilterForm`, because if we ever want to add a filterform to the question-listview …

```suggestion
class QuestionAnswerFilterForm(forms.Form):
```